src/cubing/puzzles/implementations/dynamic/side-events/2x2x2-ll.kpuzzle.svg.ts

```diff
@@ -9,18 +9,18 @@ export const cube2x2x2LLSVG = `<?xml version="1.0" encoding="UTF-8"?>
     </g>
   </defs>
   <g id="2x2x2-LL" stroke="none" stroke-width="4" style="none" stroke-linejoin="round">
-    <rect    id="CORNERS-l0-o0" stroke="#000000" style="fill: white" x="128" y="128" width="96" height="96"></rect>
-    <polygon id="CORNERS-l0-o1" stroke="#000000" style="fill: red" points="224 128 252 128 252 252 224 224"></polygon>
-    <polygon id="CORNERS-l0-o2" stroke="#000000" style="fill: limegreen" transform="translate(206, 238) scale(1, -1) rotate(-90) translate(-206, -238) " points="192 160 220 160 220 284 192 256"></polygon>
-    <rect    id="CORNERS-l1-o0" stroke="#000000" style="fill: white" x="128" y="32" width="96" height="96"></rect>
-    <polygon id="CORNERS-l1-o1" stroke="#000000" style="fill: #26f" transform="translate(206, 18) rotate(-90) translate(-206, -18) " points="192 -60 220 -60 220 64 192 36"></polygon>
-    <polygon id="CORNERS-l1-o2" stroke="#000000" style="fill: red" transform="translate(238, 50) scale(1, -1) translate(-238, -50) " points="224 -28 252 -28 252 96 224 68"></polygon>
-    <rect    id="CORNERS-l2-o0" stroke="#000000" style="fill: white" x="32" y="32" width="96" height="96"></rect>
-    <polygon id="CORNERS-l2-o1" stroke="#000000" style="fill: orange" transform="translate(18, 50) scale(-1, -1) translate(-18, -50) " points="4 -28 32 -28 32 96 4 68"></polygon>
-    <polygon id="CORNERS-l2-o2" stroke="#000000" style="fill: #26f" transform="translate(50, 18) scale(1, -1) rotate(90) translate(-50, -18) " points="36 -60 64 -60 64 64 36 36"></polygon>
-    <rect    id="CORNERS-l3-o0" stroke="#000000" style="fill: white" x="32" y="128" width="96" height="96"></rect>
-    <polygon id="CORNERS-l3-o1" stroke="#000000" style="fill: limegreen" transform="translate(50, 238) rotate(90) translate(-50, -238) " points="36 160 64 160 64 284 36 256"></polygon>
-    <polygon id="CORNERS-l3-o2" stroke="#000000" style="fill: orange" transform="translate(18, 206) scale(-1, 1) translate(-18, -206) " points="4 128 32 128 32 252 4 224"></polygon>
+    <rect    id="CORNERS-l0-o0" stroke="#000000" style="fill: white" x="128" y="128" width="88" height="88"></rect>
```

Comment:
btw, I can fix this up later, but the `stroke` should go on the surrounding `<g>` to save bytes.

Reply:
it may have gotten messed up on that site I was using